shared/src/commonMain/kotlin/org/hyperskill/app/home/injection/HomeComponentImpl.kt

```diff
@@ -33,5 +35,5 @@ class HomeComponentImpl(private val appGraph: AppGraph) : HomeComponent {
             gamificationToolbarComponent.gamificationToolbarActionDispatcher,
             topicsToDiscoverNextComponent.topicsToDiscoverNextReducer,
             topicsToDiscoverNextComponent.topicsToDiscoverNextActionDispatcher
-        )
+        ).transformState(HomeFeatureViewStateMapper::map)
```

Comment:
1. Maybe it's better to move `transformState(HomeFeatureViewStateMapper::map)` to the place where we are creating `ReduxFeature` (`HomeFeatureBuilder`), as we do with other `transformState` and `wrapWithActionDispatcher` calls?
2. Am I correct, `transformState(HomeFeatureViewStateMapper::map)` not affects `HomeFeature.State`, it only transforms what will iOS or Android clients will see? 
For example, we loaded remote data, and now `HomeFeature.State.topicsToDiscoverNextState` contains 10 topics, after `transformState(HomeFeatureViewStateMapper::map)` invoked `HomeFeature.State.topicsToDiscoverNextState` will contain 10 topics as before, but for platforms will be available only 1 topic.

Reply:
1. Yes, right remark.
2. Yes, this `transform` takes only 1st topic from topics inside `HomeFeature.State.topicsToDiscoverNextState`, and doesn't affect `HomeFeature.State` directly.

Reply:
OK, thanks!